dev-requirements.txt

```diff
@@ -1,18 +1,22 @@
-mock==1.3.0
-coverage==3.7.1
+mock==2.0.0
+coverage==4.5.1
 tox==2.9.1
-twine==1.5.0
+twine==1.11.0
 wheel==0.30.0
-tornado==4.2.1
-PySocks==1.5.6
-pkginfo>=1.0,!=1.3.0
-psutil==4.3.1
+tornado==4.5.3
+PySocks==1.6.8
+pkginfo==1.4.2
+psutil==5.4.3
 pytest-cov==2.5.1
-pytest-timeout==1.2.0
-pytest==3.4.0
-gcp-devrel-py-tools==0.0.7
+pytest-random-order==0.6.0
+pytest-timeout==1.2.1
+pytest==3.4.2
+gcp-devrel-py-tools==0.0.13
 h11==0.8.0
 # Temporary pin to work around:
 #   https://github.com/shazow/urllib3/issues/1340
 #   https://github.com/shazow/urllib3/issues/1341
```

Comment:
we can delete this comment now :-) or drop the pin altogether, idk.

There are services like requires.io and pyup.io that can automatically send PRs to update pins when never versions come out, but I don't know how fancy we want to get here.

Reply:
I removed the comment, thanks!

I think pyup.io/requires.io are a good idea, even if I never tried them. Which one would you choose?